src/Tasks/Microsoft.NET.Build.Tasks/build/platforms/Windows.targets

```diff
@@ -0,0 +1,23 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(_ShortFrameworkIdentifier)' == 'uap'">
+    <NugetTargetMoniker>UAP,Version=v10.0</NugetTargetMoniker>
+    <CopyLocalLockFileAssemblies Condition="'$(CopyLocalLockFileAssemblies)' == ''">false</CopyLocalLockFileAssemblies>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <DefineConstants Condition="'$(DisableImplicitFrameworkDefines)' != 'true'">$(DefineConstants);NETFX_CORE;WINDOWS_UWP</DefineConstants>
+    <TargetPlatformVersion Condition="'$(TargetPlatformVersion)' == ''">10.0.10240.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion Condition="'$(TargetPlatformMinVersion)' == ''">10.0.10240.0</TargetPlatformMinVersion>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
+    <LanguageTargets>$(MSBuildExtensionsPath)\Microsoft\WindowsXaml\v$(VisualStudioVersion)\Microsoft.Windows.UI.Xaml.$(_SdkLanguageName).targets</LanguageTargets>
```

Comment:
`$(MSBuildExtensionsPath)` when building using `dotnet build` is set inside of the CLI's install path. We don't expect other project types to be putting anything inside this folder.

This assumes that the "WindowsXaml" SDK is installed on the machine.  (Same for Xamarin.* below).

Instead, the intention of this new SDK (at least in my opinion) is that you shouldn't need machine specific things installed on the machine in order to build a supported project.  You should be able to install an MSBuild and a package manager (NuGet), and away you go building projects (restoring the required build assets as necessary).

This way you don't need to install VS on a build server.  Or on any machine you want to build.  It lowers the barrier of entry for people to collaborate on a project.  

`$git clone XXX`
`build`

Again this is just my opinion, but I think we should refactor these "SDK"s to align with the [MSBuild SDK vision](https://github.com/Microsoft/msbuild/issues/1493) before we take these changes into `dotnet/sdk`.